package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@digicatapult/check-version",
-  "version": "1.1.42",
+  "version": "2.0.0",
```

Comment:
As we are no longer changing default behaviour, we only need a minor version bump now :)